tokio/src/process/mod.rs

```diff
@@ -997,10 +997,14 @@ impl Child {
     /// ```no_run
```

Comment:
It might be good to remove the `no_run` so the test actually runs. You should be able to disable it on windows with a `#[cfg(unix)]` or similar.

Reply:
That is a fair point, is there a reason for leaving the tests out?